dsp/impulse/index.js

```diff
@@ -0,0 +1,25 @@
+export const impulse = async(context) => {
+  var pulse = 0;
+  return ({n}) => {
+    pulse = (n === 0 ? 1 : 0);
+    return pulse;
+  }
+}
+
+
+export const dust = async(context, density) => {
+  var pulse;
+  var coin;
+  var dens = density ;
```

Comment:
I think we can pass the `density` variable directly to line 18, so this line isn't necessary. Also the line above, `coin` isn't used anywhere below so that can be removed also.